public/content/lessons/2016/eigenvalues/index.mdx

```diff
@@ -7,4 +7,1048 @@ video: PFDu9oVAE-g
+
+<Figure
+    image="./figures/working-in-an-eigenbasis/BothBasisVectorsAreEigenvectors.light.svg" 
+    video="./figures/working-in-an-eigenbasis/BothBasisVectorsAreEigenvectors.light.mp4" 
+    width="512px"
+/>
+
+Any time a matrix has zeros everywhere other than the diagonal, it's called a "diagonal matrix", and the way to interpret it is that all the basis vectors are eigenvectors, and the diagonal entries of the matrix give you their corresponding eigenvalues.
+
+$$
+\left[\begin{array}{cccc}
+-5 & 0 & 0 & 0 \\
+0 & -2 & 0 & 0 \\
+0 & 0 & -4 & 0 \\
+0 & 0 & 0 & 4
+\end{array}\right]
+$$
+
+There are a number of things that make diagonal matrices nicer to work with. One big one is that it's easier to compute what will happen if you multiply this matrix by itself a whole bunch of times.
+
+$$
+\begin{aligned}
+\left[\begin{array}{ll}
+3 & 0 \\
+0 & 2
+\end{array}\right]
+\left[\begin{array}{l}
+x \\
+y
+\end{array}\right]
+& =
+\left[\begin{array}{c}
+3 x \\
+2 y
+\end{array}\right] \\ \rule{0pt}{2.5em}
+
+\left[\begin{array}{ll}
+3 & 0 \\
+0 & 2
+\end{array}\right]
+\left[\begin{array}{c}
+3 x \\
+2 y
+\end{array}\right]
+& =
+\left[\begin{array}{c}
+3^2 x \\
+2^2 y
+\end{array}\right] \\ \rule{0pt}{2.5em}
+
+\left[\begin{array}{ll}
+3 & 0 \\
+0 & 2
+\end{array}\right]
+\left[\begin{array}{c}
+3^2 x \\
+2^2 y
+\end{array}\right]
+& =
+\left[\begin{array}{c}
+3^3 x \\
+2^3 y
+\end{array}\right] \\ \rule{0pt}{2.5em}
+
+& \ldots 
+\end{aligned}
+$$
+
+Since all one of these matrices does is scale each basis vector by some eigenvalue, applying that matrix, say, 100 times will just correspond to scaling each basis vector by the 100th power of the corresponding eigenvalue.
+
+$$
+
+\underbrace{\left[\begin{array}{ll}
+3 & 0 \\
+0 & 2
+\end{array}\right] \cdots \left[\begin{array}{ll}
+3 & 0 \\
+0 & 2
+\end{array}\right]\left[\begin{array}{ll}
+3 & 0 \\
+0 & 2
+\end{array}\right]\left[\begin{array}{ll}
+3 & 0 \\
+0 & 2
+\end{array}\right]}_{100 \text { times }}\left[\begin{array}{l}
+x \\
+y
+\end{array}\right]
+= 
+\left[\begin{array}{cc}
+3^{100} & 0 \\
+0 & 2^{100}
+\end{array}\right]\left[\begin{array}{l}
+x \\
+y
+\end{array}\right]
+
+$$
+
+
+In contrast, try to compute the 100th power of a non-diagonal matrix would be. Really, think about it for a moment, it'd be a nightmare.
+
+### Change of basis
+
+Of course, you will rarely be so lucky as to have your basis vectors also be eigenvectors. But if your transformation has lots of eigenvectors, like the one from the start of the video, enough that you can choose a set that spans the full space. Then you can change your coordinate system so that these eigenvectors are your basis vectors.
```

Comment:
```suggestion
Of course, you will rarely be so lucky as to have your basis vectors also be eigenvectors. But if your transformation has lots of eigenvectors, enough that you can choose a set that spans the full space, then you can change your coordinate system so that these eigenvectors are your basis vectors.
```